src/fmeval/transforms/semantic_perturbations.py

```diff
@@ -0,0 +1,296 @@
+import numpy as np
+from typing import Dict, List, Any
+from abc import abstractmethod
+
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.util import validate_call
+from fmeval.util import require
+
+
+class SemanticPerturbation(Transform):
```

Comment:
why are we not converting the existing `SemanticPerturbationUtil` to a transform? Won't it be cleaner? This is leading to a confusing commit history.

Reply:
Since I wanted this PR to be small, and focused on implementing the new transforms (and not for updating the eval algos that use semantic perturbations), I did not get rid of `SemanticPerturbationUtil`, which was being relied on by all of the semantic robustness algos.